starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,69 @@
+html{
+  margin: -20px 0px;
+}
+
+body{
+  text-align: center;
+font-family: Helvetica;
+border: 0px;
+padding: 0px 20px 30px;
+margin: 0;
+font-weight: lighter;
+}
+.H1{
+  background-image:url("/Users/ivanjorge/Prework/code/labs/lab-css-recipes-clone/starter-code/apple-pie-starter-code/images/apple-pie.jpg");
```

Comment:
try using a relative path for this background image though.  Using an absolute path won't work on my computer because I don't have a file "/Users/ivanjorge/Prework/code/labs/lab-css-recipes-clone/starter-code/apple-pie-starter-code/images/apple-pie.jpg" But I do have a file "./images/applie-pie.jps"